servers/Azure.Mcp.Server/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The Azure MCP Server updates automatically by default whenever a new release com
 
 ### Features Added
 
+- Added Azure Storage Sync (StorageSync) module with 24 commands for managing cloud synchronization of file shares:
```

Comment:
I see 18 commands and not 24 in the code files.

Reply:
Commands found:

StorageSyncServiceCreateCommand
StorageSyncServiceDeleteCommand
StorageSyncServiceGetCommand
StorageSyncServiceUpdateCommand
RegisteredServerGetCommand
RegisteredServerUnregisterCommand
RegisteredServerUpdateCommand
SyncGroupCreateCommand
SyncGroupDeleteCommand
SyncGroupGetCommand
CloudEndpointCreateCommand
CloudEndpointDeleteCommand
CloudEndpointGetCommand
CloudEndpointTriggerChangeDetectionCommand
ServerEndpointCreateCommand
ServerEndpointDeleteCommand
ServerEndpointGetCommand
ServerEndpointUpdateCommand